docs/configuring-a-resource.md

```diff
@@ -376,6 +489,50 @@ message in its `status.conditions`, we do the `LateInitializer.IgnoreFields`
 custom configuration detailed above to skip one of the mutually exclusive fields
 during late-initialization.
 
+### Overriding Terraform Resource Schema
+
+Terrajet generates Crossplane resource schemas (CR spec/status) using the
+[Terraform schema of the resource]. As of today, Terrajet leverages the
+following attributes in the schema:
+
+- [Type] and [Elem] to identify the type of the field.
+- [Sensitive] to see if we need to keep it in a Secret instead of CR.
+- [Description] to add as a description to the field in CRD.
+- [Optional] and [Computed] to identify whether the fields go under spec or
+status:
+  - Not Optional & Not Computed => Spec (required)
+  - Optional & Not Computed => Spec (optional)
+  - Optional & Computed => Spec (optional, to be late-initialized)
+  - Not Optional & Computed => Status
+
+Usually, we don't need to make any modifications in the resource schema and
+resource schema just works as is. However, there could be some rare edge cases
+like:
+
+- Field contains sensitive information but not marked as `Sensitive` or vice
```

Comment:
Addition of `region` field is also another example.

Reply:
Added as yet another example.